processing/pom.xml

```diff
@@ -44,6 +44,10 @@
             <artifactId>emitter</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.metamx</groupId>
```

Comment:
@fjy we don't exclude it anywhere else (see pom in druid-server module), so why would we need to exclude it here?
It's marked as optional in server-metrics, so if you want sigar you still have to include it explicitly in your own build.